stages/org.osbuild.systemd

```diff
@@ -40,16 +53,75 @@ SCHEMA = """
   "default_target": {
     "type": "string",
     "description": "The default target to boot into"
+  },
+  "unit_dropins": {
+    "additionalProperties": false,
+    "type": "object",
+    "description": "Systemd unit drop-in configurations.",
+    "patternProperties": {
+      "^[\\w:.\\\\-]+[@]{0,1}[\\w:.\\\\-]*\\.service$": {
+        "additionalProperties": false,
+        "type": "object",
+        "description": "Drop-in configurations for a '.service' unit.",
+        "patternProperties": {
+          "^[\\w.-]{1,250}\\.conf$": {
+            "additionalProperties": false,
+            "type": "object",
+            "description": "Drop-in configuration for a '.service' unit.",
+            "properties": {
+              "service": {
```

Comment:
Fine as is, but we could also capitalise it, since that is the correct name in the unit file. I am doing that in the `oci-archive` stage where the JSON keys follow the OCI naming (and are capitalised).

Reply:
I'm totally fine with that. I just followed the naming already used in the stage and other stages as I was not sure if there are any guidelines for this.

Reply:
I made the change. Also noticed one thing that I missed - to override ` ConfigParser.optionxform(option)` to prevent modification of the option name to lowercase. This is fixed now.